primitives/runtime/src/generic/digest.rs

```diff
@@ -118,6 +118,14 @@ pub enum DigestItem<Hash> {
 
 	/// Some other thing. Unsupported and experimental.
 	Other(Vec<u8>),
+
+	/// An indication for the light clients that the runtime execution
+	/// environment is updated.
+	///
+	/// Currently this is triggered when:
+	/// 1. Runtime code blob is changed or
+	/// 2. `heap_pages` value is changed.
+	RuntimeUpdated,
```

Comment:
I personally wouldn't mind the more accurate name `RuntimeOrHeapPagesUpdated`

Reply:
`RuntimeEnvironmentUpdated`